view/adminhtml/templates/common.phtml

```diff
@@ -312,15 +37,15 @@ document.addEventListener("DOMContentLoaded", function(event) {
 
           // Only check if the focus is not on the search
           if ($(':focus').attr('name') != 'query_text') {
-              new Ajax.Request(
-                url,
-                {
-                    parameters: {'store_id': storeId, 'query_text': value, 'query_id': queryId},
-                    method: 'post',
-                    asynchronous: false,
-                    onComplete: function (transport) {
-                    var response = JSON.parse(transport.responseText);
-                    isValid = response.isValid;
+            new Ajax.Request(
```

Comment:
@damcou Is the reason why we are keeping prototype for this Ajax class? Wouldn't be better to convert this to jquery's Ajax? https://api.jquery.com/jquery.ajax/